.docker/install-plugin-theme.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Install plugins
+echo "Installing plugins"
+wp --allow-root --url=wordcamp.test plugin install akismet buddypress bbpress jetpack wp-multibyte-patch wordpress-importer polldaddy liveblog wp-super-cache custom-content-width camptix-network-tools email-post-changes tagregator supportflow camptix-pagseguro camptix-payfast-gateway camptix-trustpay camptix-trustcard camptix-mercadopago camptix-kdcpay-gateway campt-indian-payment-gateway camptix
```

Comment:
`MUST` - CampTix should probably be cloned from GitHub, since we'll want the development version.

IIRC the Meta Environment provision script does that with a few others too.

Reply:
I thought about this, but I guess responsibility of VCS for anything inside `wp-content` folder should be on host instead of Docker 🤔 . So if someone wants to develop on CampTix, they should ideally replace the directory inside `wp-content/plugin` folder. What do you think?

Reply:
🤔 , I thought that Docker shared the folder between the host and container, so that they're essentially the same? Given that, I'm not sure why the responsibility would shift?

It seems like we'd just be adding an extra step that has to be done manually, rather than automating it.

Reply:
Docker does shared folder, however VCS tools like Git or SVN are not guaranteed to be installed inside it.
I visualize Docker setup like a run time environment, so everything related to running the site should be inside the Docker, but source control should be outside it. For egs, someone can download the source code zip and be able to run Docker, without having to setup or log in to Git, SVN or any other tool which is not required to run it.
Since CampTix is also open source, there is no blocker to actually clone it from GitHub although I feel the setup is much cleaner because of this distinction, but that's just a preference.

Reply:
🤔 yeah, I can see that point of view. I personally still lean towards the view that the practical goal is more important than maintaining purity.

Another way to archive the goal would be to make CampTix a Git submodule, but Git submodules are pretty awful. 

If there are any other steps like this, then maybe another solution would be to provide a script that the user could run post-installation in order to replace the downloaded code w/ a cloned repo, etc. Maybe not worth it if this is the only thing, though.

🤷‍♂ , not a big deal either way. Go with whatever feels best to you.

Reply:
For now lets keep these separate. If people complain about this, then we will revisit and make the change. Resolving the conversation for now